eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -403,6 +403,17 @@ stages:
         targetOS: windows
         targetArchitecture: x64
 
+    - template: ../jobs/vmr-build.yml
+      parameters:
+        buildName: Windows_BuildTests
+        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+        vmrBranch: ${{ variables.VmrBranch }}
+        pool: ${{ parameters.pool_Windows }}
+        targetOS: windows
+        targetArchitecture: x64
+        extraProperties: /p:DotNetBuildTests=true
+        runTests: false
+
```

Comment:
If this doesn't take longer than 120 minutes we can keep the leg enabled in PR validation but otherwise we would need to move it out.

Reply:
OK - I will update the PR to move this job up a bit to this location: https://github.com/dotnet/sdk/blob/0bc10be7a8a9b1ce7b762d345af883d4b6bedd14/eng/pipelines/templates/stages/vmr-build.yml#L376

I'll also remove the `sign` parameter that I just added when I was resolving the merge conflicts.